qa/tasks/vstart_runner.py

```diff
@@ -940,6 +942,19 @@ def clear_firewall(self):
         pass
 
     def newfs(self, name='cephfs', create=True):
+        fs_ls_output = self.mon_manager.run_cluster_cmd(
+            args='fs ls', stdout=StringIO()).stdout.getvalue()
+
+        fsname = name
+        while fsname in fs_ls_output:
+            fsname = 'cephfs_' + choice(ascii_lowercase) + choice(digits)
+
+        if fsname != name:
+            log.info(f'changing fs name from "{name}" to "{fsname}" because '
+                     f'a Ceph FS named "{name}" is already present in the '
+                      'cluster.')
+            name = fsname
```

Comment:
what circumstances cause this?

Reply:
`name` parameter has a default value. So, not passing a value for parameter `name` to `Filesystem.newfs()` will make the method attempt to create a Ceph FS named `cephfs` which will make the test fail if `cephfs` already exists on the cluster. So, it's better to have few lines of code that checks if `cephfs` exists already and pick a new string if it does.

We usually pick and hardcode FS names in our tests which won't be a necessity after this. Passing no parameters to `self.fs.newfs()`, would also be fine.

Reply:
But I don't understand what situation (i.e. existing test) will cause `newfs` to be called such that the `cephfs` file system already exists. That would be a bug. In which case this code should never run.

Reply:
Why not just name it when you are creating the fs2 ?

Reply:
> Why not just name it when you are creating the fs2 ?

I can really circumvent this by doing that but since it is possible to automate this and it would be better to automate this, why not.

Reply:
Having static names (especially names unique to a test) is helpful for grepping logs.

I think we should drop this one but it was a good initiative on your part.